app/tag/tag-list-directive.html

```diff
@@ -7,10 +7,16 @@
 </button>
 
 <div class="table-responsive">
-  <table class="table table-hover">
+  <table st-table="displayedTags" st-safe-src="tags" class="table table-hover">
```

Comment:
As I was in the v1, I just let it "as is". And not sorting is not working with my pull request so I am going to remove thoses attributes and I'll dig into this later